.circleci/unittest/linux/scripts/install.sh

```diff
@@ -52,13 +52,13 @@ printf "Installing PyTorch with %s\n" "${cudatoolkit}"
     if [[ -z "$cudatoolkit" ]]; then
         conda install ${CONDA_CHANNEL_FLAGS:-} -y -c "pytorch-${UPLOAD_CHANNEL}" $MKL_CONSTRAINT "pytorch-${UPLOAD_CHANNEL}::${pytorch_build}"
     else
-        conda install pytorch ${cudatoolkit} ${CONDA_CHANNEL_FLAGS:-} -y -c "pytorch-${UPLOAD_CHANNEL}" -c nvidia  $MKL_CONSTRAINT
+        conda install pytorch ${cudatoolkit} cuda-toolkit=${CUDA_VERSION} cuda-nvcc=${CUDA_VERSION} cuda-compiler=${CUDA_VERSION} cuda-driver-dev=${CUDA_VERSION} ${CONDA_CHANNEL_FLAGS:-} -y -c "pytorch-${UPLOAD_CHANNEL}" -c nvidia  $MKL_CONSTRAINT
```

Comment:
A nice tip is instead of using e.g. `conda install -c nvidia cudatoolkit=11.8.0`, instead use `conda install -c nvidia/label/cuda-11.8.0 cudatoolkit`. This ensures it will never pull in a dependency with the wrong CUDA version.

Reply:
Thanks for the tip. Let me try.

Reply:
@mthrok Nice! Thanks Moto!

Reply:
@mthrok Try to include all these dependencies that we pruned from pytorch-cuda package:
https://github.com/pytorch/builder/commit/942c85481eb9f3c67aafd21bb3292202d4f1c4ef
It should pick up all the required libraries